### ISSUE TITLE
Two related ERRORS from WCAG 2.0 that should be fixed    (Notes on 1.3.3 and 1.4.1)

### ISSUE THREAD

**Issue by GreggVan** [w3c/wcag21#280]
I KNOW THAT THE NOTE ON 1.3.3 CAME FROM WCAG 2.0 ---   But the notes on 1.3.3. and 1.4.1 are wrong.  They were added in response to a comment - but the commenter was confused (or correct) and the WG became confused and messed this up.     

1.3.3 and 1.4.1  are totally different SC that cover totally different concepts. 

1.3.3. is about the **words in the instructions** it has nothing to do with guideline 1.4 which has to do with the color **of the text** that is used to create the instructions.   1.3.3. is talking about the fact that you shouldn't use sensory information (eg COLOR or other WORDS)  when writing the text of instructions. 

1.4.1  has to do with the use of actual color to convey information (**not color words** like "blue" **but the actual color** blue).  

RECOMMENDED RESOLUTION:   
1.	Notes should be deleted from both SC.   
2.	Add the word “color” to the list of sensory characteristics in SC1.3.3.   It is actually already covered there – but not explicitly -  and it should be added so that it is explicitly and unambiguously covered here.

This is actually an ERRATA level correction since it does not change either SC but removes the confusion.

**Comment by michael-n-cooper** [comment]
This is an issue on WCAG 2.0, since in WCAG 2.1 it just inherited these notes. We should process it on WCAG 2.0 first, then propagate the disposition to WCAG 2.1. Therefore, I have filed [WCAG 2.0 Comment 267](https://github.com/w3c/wcag/issues/267) referencing this issue, and we should hold further processing of this in WCAG 2.1 until it's closed in WCAG 2.0.

**Comment by awkawk** [comment]
This was addressed and is integrated into the WCAG 2.0 errata: https://www.w3.org/WAI/WCAG20/errata/
